.github/workflows/stale.yml

```diff
@@ -10,9 +10,17 @@ jobs:
       - uses: actions/stale@v8.0.0
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # PR's
           stale-pr-message: "This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions."
           days-before-stale: -1
           days-before-close: -1
           days-before-pr-stale: 45
           days-before-pr-close: 6
+          close-pr-label: "stale"
           exempt-pr-labels: "pinned, security, proposal, blocked, ADR"
+          # Issues
+          days-before-issue-stale: 14
```

Comment:
In short, there will be 20 days between the last message on the issue and the issue being closed.

Issues can be re-opened if new discussion arises.

Can other members start a discussion after the issue is closed? It would be great if they could still write a message and tag us it so we could get notified about new activity.

Reply:
Yep closed issues can be commented on and its possible to @ people. This works for non-Cosmos org accounts too.